### PR TITLE
Improve custom active response configuration section and add some examples

### PR DESCRIPTION
**Pull request by cborla**
Closes https://github.com/wazuh/wazuh-documentation/issues/4287

Custom Active Response configuration section and some examples.

## Description
Explained all the necessary steps to configure a custom active responses.

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns.

### CODE REVIEW

**Review by MarVergara — APPROVED** [review]
LGTM on the editorial side.

**Review by TomasTurina — APPROVED** [review]
LGTM!

File: source/user-manual/capabilities/active-response/custom-active-response.rst [hunk 1]
```diff
@@ -65,81 +162,414 @@ The response message from execd is a follows:
...
+
+        <command>
+            <name>custom-ar</name>
+            <executable>custom-ar.exe</executable>
+            <timeout_allowed>yes</timeout_allowed>
+        </command>
+
+With this setting, you are able to run an application instead of a Python script when triggering an active response on a Windows system.
+
+Run a Python script through a Bash launcher
```
Comment (MarVergara) [inline source/user-manual/capabilities/active-response/custom-active-response.rst:454]:
```suggestion
Run a Python script through a Batch launcher
```

File: source/user-manual/capabilities/active-response/custom-active-response.rst [hunk 2]
```diff
@@ -65,81 +162,414 @@ The response message from execd is a follows:
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (MarVergara) [inline source/user-manual/capabilities/active-response/custom-active-response.rst:455]:
```suggestion
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

File: source/user-manual/capabilities/active-response/custom-active-response.rst [hunk 3]
```diff
@@ -5,53 +5,150 @@
...
+            },
+            "program":"/var/ossec/active-response/bin/firewall-drop"
+        }
     }
-  }
 
+Stateful active responses
+-------------------------
+
+A Stateful AR undoes its original action after the period of time specified in the active response. That is to say, the stateful AR performs the basic actions of a stateless AR to later undo the process based on the command configuration. As part of the timeout behavior, when the received command is ``add``, the AR must execute this operation.
```
Comment (s-ocando) [inline source/user-manual/capabilities/active-response/custom-active-response.rst:111]:
```suggestion
A stateful AR undoes its original action after the period of time specified in the active response. That is to say, the stateful AR performs the basic actions of a stateless AR to later undo the process based on the command configuration. As part of the timeout behavior, when the received command is ``add``, the AR must execute this operation.
```

**Review by s-ocando — APPROVED** [review]

**Pull request merged**